src/components/chips/js/chipsController.js

```diff
@@ -397,6 +397,7 @@ MdChipsCtrl.prototype.hasMaxChipsReached = function() {
  */
 MdChipsCtrl.prototype.validateModel = function() {
   this.ngModelCtrl.$setValidity('md-max-chips', !this.hasMaxChipsReached());
+  this.ngModelCtrl.$validate(); // rerun any registered validators
```

Comment:
Looking at the angular documentation it appears to have been added in [1.3.0](https://code.angularjs.org/1.3.0/docs/api/ng/type/ngModel.NgModelController), which is the minimum version angular material supports.